.github/workflows/README.md

```diff
@@ -23,7 +23,7 @@ Individual workflows with changes:
 - `ci_javascript.yml`: Runs tests for the JS files. Tests must run from the project root folder. You'll need to install NodeJS and the JS dependencies:
 
 ```yml
-- uses: actions/setup-node@master
+- uses: actions/setup-node@main
```

Comment:
Why are still using Main here and not the V3 dirrectly ?

Reply:
This is for the documentation, I thought there might be some reason why it was set to `@master` before.

Didn't think about it further, just updated it to the correct branch.